coforma/swift-tech-challenge
# Fixed bug with templates, revved some other documentaiton as well

# Bug in user templates
Fixes #6 
Related to #1 

# Summary

Adds a real link to the issue templates for bugs and user stories, adds a PR template that was left out of previous commit as well as gitignore file

## Type of change

Please delete options that are not relevant.


# How To Test
1. Create new issue on board
2. Verify definition of done link to work item is legit
3. Delete test item

# Checklist:
